dpdata/abacus/md.py

```diff
@@ -97,53 +99,47 @@ def get_coords_from_cif(ndump, dump_freq, atom_names, natoms, types, path_out, c
         pos_file = os.path.join(path_out, "md_pos_%d.cif" %(dump_idx*dump_freq))
         #print("dump_idx = %s" %dump_idx)
         coords[dump_idx] = get_single_coord_from_cif(pos_file, atom_names, natoms, cell)
-    return coords
+    return coords'''
```

Comment:
Please REMOVE the lines if you do not need them anymore. Do NOT use the doc string to comment out the code.

Reply:
> Please REMOVE the lines if you do not need them anymore. Do NOT use the doc string to comment out the code.

Sorry for that. I will fix it.